src/test/resources/application-test.yml

```
@@ -1,4 +1,16 @@
   data:
     neo4j:
-      database: neo4j
\ No newline at end of file
+      database: neo4j
+
+---
+spring:
+  config:
+    import: classpath:application-dev.properties
+  main:
+    allow-bean-definition-overriding: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)